pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -972,4 +974,11 @@ public static class Range {
   public static class IdealState {
     public static final String HYBRID_TABLE_TIME_BOUNDARY = "HYBRID_TABLE_TIME_BOUNDARY";
   }
+
+  public static class RewriterConstants {
+    public static final String PARENT_AGGREGATION_NAME_PREFIX = "pinotparentaggregation";
```

Comment:
(nit) Probably don't need `Pinot` as a prefix ?. We don't have any functions named that way

Reply:
removed